### PR TITLE
Moogsoft Shim

### PR DESCRIPTION
**Pull request by johnddias**
Shim written based on customer requirement.  Customer wanted recommendations included in the payload to Moogsoft so this shim also does call back to vROps to look up additional information not included in the alert message.

There are also fields required by Moogsoft that aren't provided by LI natively (sev level, resource ID, alert state) and thus integration with this shim should be done by having LI forward alerts to vROps.

### CODE REVIEW

**Comment by johnddias** [comment]
Omit for this shim because I don't have a standing moogsoft instance to test against.

**Comment by johnddias** [comment]
Please do not merge yet, customer had some additional requests that I'm adding over the weekend.  Will update when I'm ready - thanks!!

**Comment by johnddias** [comment]
OK, this is ready for review/approval - thanks!!

File: loginsightwebhookdemo/__init__.py [hunk 1]
```diff
@@ -141,7 +141,9 @@ def parsevROps(payload, alert):
         return alert
     alert.update({
         "hookName": "vRealize Operations Manager",
+        ""
```
Comment (alanjcastonguay) [inline loginsightwebhookdemo/__init__.py]: This is unlikely to be intentional. @johnddias please review.

**Comment by johnddias** [comment]
Good catch, corrected. @alanjcastonguay

**Comment by alanjcastonguay** [comment]
I'm concerned that the above error was not caught by tests.

**Pull request merged**